x-pack/plugins/transform/server/routes/api/transforms.ts

```diff
@@ -174,14 +183,43 @@ export function registerTransformsRoutes(routeDependencies: RouteDependencies) {
       path: addBasePath('delete_transforms'),
       validate: {
         body: schema.maybe(schema.any()),
+        query: schema.object({
+          /**
+           * Analytics Destination Index
```

Comment:
Should say `Transform Destination Index`

Reply:
Fixed in https://github.com/elastic/kibana/pull/67922/commits/d3c37b5c0d03c2ca6e211a3ff2380154223c2f0d